terraform/shared/modules/env/postgrest.tf

```diff
@@ -16,7 +16,7 @@ resource "cloudfoundry_service_key" "postgrest" {
 resource "cloudfoundry_app" "postgrest" {
   name         = local.postgrest_name
   space        = data.cloudfoundry_space.apps.id
-  docker_image = var.postgrest_image
+  docker_image = "ghcr.io/gsa-tts/fac/postgrest:latest"
```

Comment:
(Non-Blocking)

We can use YYYYMMDD if we want here, however, the assumption is that `:latest` will always be a "clean" image, as in order for it to get into our GHCR, it needs to have 0 scanned items via trivy.

Reply:
[Same as above](https://github.com/GSA-TTS/FAC/pull/1520#discussion_r1267085468)